views/index.py

```diff
@@ -1,7 +1,65 @@
-from flask import render_template
+from flask import render_template, request, session
 from . import library
+from models.user import User
+from forms.login_form import LoginForm
+from forms.registration_forms import RegistrationForm
+from werkzeug.security import generate_password_hash, check_password_hash
+from app import db
+from flask_login import LoginManager
+
+login_manager = LoginManager()
 
 
 @library.route('/')
 def index():
     return render_template('index.html')
+
+
+# @login_manager.user_loader
+# def load_user(user_id):
+#     return user_id
+
+
+@library.route('/login', methods=['GET', 'POST'])
+def login():
+    if request.method == 'GET':
+        form = LoginForm()
+        return render_template('login.html', form=form)
+    else:
+        form = LoginForm()
+        try:
+            if form.validate():
+                data = User.query.filter_by(email=form.email.data).first()
+                if data is not None and check_password_hash(data.password_hash, form.password.data):
+                    session['logged_in'] = True
+                    session['id'] = data.id
+                    session['email'] = data.email
+                    print(session)
```

Comment:
Eliminate print from source code. To debug better use pdb: [https://docs.python.org/3/library/pdb.html](url)